contrib/win32/win32compat/w32-sshfileperm.c

```diff
@@ -40,6 +40,8 @@
 #include "misc_internal.h"
 #include "config.h"
 
+extern int log_on_stderr;
```

Comment:
since `BOOL` is already used elsewhere, why is this an `int`?

Reply:
It's from upstream code - https://github.com/PowerShell/openssh-portable/blob/latestw_all/log.c.
I pulled it in here to ensure the logging message goes to event viewer rather than stderr.